packages/commonwealth/client/scripts/helpers/tooltipTexts.ts

```diff
@@ -0,0 +1,15 @@
+import { ChainBase } from '@hicommonwealth/core';
+
+export const disabledStakeButtonTooltipText = ({
+  isLoggedIn = true,
+  connectBaseChainToBuy,
+}: {
+  isLoggedIn?: boolean;
+  connectBaseChainToBuy?: ChainBase;
+}) => {
```

Comment:
I don't feel strongly about it but it can usually be easier to define a type for the args but it's just a style thing.

Reply:
done